index.html

```diff
@@ -0,0 +1,578 @@
+    </aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span>
+  </code>
+</figure>
+
+The final two segments of the URL's [path](https://tools.ietf.org/html/rfc3986#section-3.3) **should** contain the spec's [default prefix](#prefix) and a [version tag](#versioning). The content of the URL up to and including the prefix—but excluding the version tag and trailing `/`—is the spec's *identity*. For the above example,
+<dl>
+  <dt markdown>`identity: "https://spec.example.com/a/b/c/exampleSpec"`</dt>
+  <dd>A global identifier for the spec. Processors can treat this as an opaque string identifying the spec (but not the version of the spec) for purposes of selecting an appropriate implementation.</dd>
+  <dt markdown>`name: "exampleSpec"`</dt>
+  <dd markdown>The spec's name, for purposes of [prefixing](#prefixing) schema elements it defines or extends.</dd>
+  <dt markdown>`version: "v1.0"`</dt>
+  <dd markdown>The tag for the [version](#versioning) of the spec used to author the document. Processors **must** select an implementation of the spec which can [satisfy](#satisfaction) the specified version.</dd>
+</dl>
+
+The version tag **must** satisfy the [BNF grammar for `‹version tag›`](#backus-naur-form-for-versions-and-version-tags).
+
+### Ignore meaningless URL components
+
+When extracting the URL's `prefix` and `version`, processors **must** ignore any url components which are not assigned a meaning. This spec assigns meaning to the final two components of the [path](https://tools.ietf.org/html/rfc3986#section-3.3). Other URL components—particularly query strings and fragments, if present—**must** be ignored for the purposes of extracting the `prefix` and `version`.
+
+<figure>
+  <figcaption>
+    <a name=spec-url-ignore-meaningless href=#spec-url-ignore-meaningless class=a-header>Ignoring meaningless parts of a URL</a>
+  </figcaption>  
+  <code class=anatomy>
+    <span class=pink style='--depth: 2'>https://example.com/<span>exampleSpec<aside>name</aside></span><aside>
+      identity
+    </aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span><span class=grey>?key=val&k2=v2#fragment<aside>ignored</aside></span>
+  </code>
+</figure>
+
+### Why is versioning in the URL, not a directive argument?
+
+The version is in the URL because when a human reader visits the URL, we would like them to be taken to the documentation for the *version of the specification in use by this document*. Many text editors will turn URLs into hyperlinks, and it's highly desirable that clicking the link takes the user to the correct version of the docs. Putting the version information in a separate argument to the `@core` directive would require this.
+
+## Prefixing
+Specifications meant to be referenced via `@core` **should** prefix any schema elements they introduce. The prefix:
+1. **must** be a string of characters valid within GraphQL typenames, with the exception of two underscores (`"__"`)
```

Comment:
s/the exception/no substring consisting/ ?

Also I think we should forbid ending in underscores.  Otherwise you can do something like `@core(using: "...", as: "foo") @core(using: "...", as: "foo_")` and wonder whether `foo___bar` is `foo`'s `_bar` or `foo_`'s `bar`.  I'm not sure if there's ambiguity from allowing leading underscores but they might be worth banning too.